### ISSUE TITLE
contribution ....

### ISSUE THREAD

**Issue by roblatour** [mbadolato/iTerm2-Color-Schemes#350]
thanks for posting all this...

Here is a contribution for Windows Terminal, looks great with font 'Consolas'

      {
            "background": "#000000",
            "black": "#13A10E",
            "blue": "#13A10E",
            "brightBlack": "#16BA10",
            "brightBlue": "#16BA10",
            "brightCyan": "#16BA10",
            "brightGreen": "#16C60C",
            "brightPurple": "#16BA10",
            "brightRed": "#16BA10",
            "brightWhite": "#16BA10",
            "brightYellow": "#16BA10",
            "cursorColor": "#16BA10",
            "cyan": "#13A10E",
            "foreground": "#16BA10",
            "green": "#13A10E",
            "name": "Retro",
            "purple": "#13A10E",
            "red": "#13A10E",
            "selectionBackground": "#FFFFFF",
            "white": "#13A10E",
            "yellow": "#13A10E"
        }

tweak it if you like

**Comment by mbadolato** [comment]
Hi, thanks! I do need it provided in itermcolors format though, which is then used as the basis to create the other versions. Any chance you could provide that? I have no way currently to convert from another format to the others. It would also need a screenshot, credits, etc (per the Contirbuting doc)

**Comment by roblatour** [comment]
Hi, it looks like you need a Mac to create something in the itermcolors format, something sadly I don't have.   

I posted here because of the link from https://windowsterminalthemes.dev/ which does say to use the itermcolors format, but again no Mac for me so I gave it a try with what I had. 

Thanks all the same.

**Comment by mbadolato** [comment]
Much appreciated; maybe someone here will look at the color defs and put together a PR with the iterm format

**Comment by jdhmtl** [comment]
Haven't been around much of late, but I'm happy to take care of this.

**Comment by jdhmtl** [comment]
Looks like there are only two colours in this scheme. Does this image look more or less correct, @roblatour?

<img width="640" alt="Retro" src="https://user-images.githubusercontent.com/3738327/151611600-0c37dd52-3eb9-4549-8e25-92822108ed6d.png">

**Comment by roblatour** [comment]
Hi John,

Yes - that's exactly it, and why I named it 'Retro'.

It harks back to the days when certain terminals would only have green characters.  They were surprising easy on the eyes for working with over long periods.

The highlighting of selected text, something you couldn't do back in the day, is however not worth passing on and I was glad to see it could be supported.

Here is what I mean:

***@***.***

In setting this up I also experimented with some publicly available dot matrix fonts.  However, found the one I ended up using, Consolas, to be better while still keeping to the 'Retro' look.

Were you able to find a way to get this available for use by others?

Rob

**Comment by roblatour** [comment]
Hi John,

Yes - that's exactly it, and why I named it 'Retro'.

It harks back to the days when certain terminals would only have green characters.  They were surprising easy on the eyes for working with over long periods.

The highlighting of selected text, something you couldn't do back in the day, is however not worth passing on and I was glad to see it could be supported.

Here is what I mean:



In setting this up I also experimented with some publicly available dot matrix fonts.  However, found the one I ended up using, Consolas, to be better while still keeping to the 'Retro' look.

Were you able to find a way to get this available for use by others?

Rob
